[2024-10-16T09:55:37.467+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-05-20T00:00:00+00:00 [queued]>
[2024-10-16T09:55:37.477+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-05-20T00:00:00+00:00 [queued]>
[2024-10-16T09:55:37.478+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T09:55:37.478+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2024-10-16T09:55:37.479+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T09:55:37.494+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): insert_rate> on 2024-05-20 00:00:00+00:00
[2024-10-16T09:55:37.500+0000] {standard_task_runner.py:55} INFO - Started process 8760 to run task
[2024-10-16T09:55:37.505+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'exchange_rate_usd_kzt_dag', 'insert_rate', 'scheduled__2024-05-20T00:00:00+00:00', '--job-id', '1757', '--raw', '--subdir', 'DAGS_FOLDER/exchange_rates/__init__.py', '--cfg-path', '/tmp/tmp6ielol05']
[2024-10-16T09:55:37.507+0000] {standard_task_runner.py:83} INFO - Job 1757: Subtask insert_rate
[2024-10-16T09:55:37.591+0000] {task_command.py:389} INFO - Running <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-05-20T00:00:00+00:00 [running]> on host 63ee4fa64e74
[2024-10-16T09:55:37.634+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:205: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2024-10-16T09:55:37.686+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=exchange_rate_usd_kzt_dag
AIRFLOW_CTX_TASK_ID=insert_rate
AIRFLOW_CTX_EXECUTION_DATE=2024-05-20T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-05-20T00:00:00+00:00
[2024-10-16T09:55:37.687+0000] {sql.py:253} INFO - Executing: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'KZT', 443.83834098, '2024-05-20')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;
[2024-10-16T09:55:37.697+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T09:55:37.914+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T09:55:37.922+0000] {sql.py:364} INFO - Running statement: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'KZT', 443.83834098, '2024-05-20')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;, parameters: None
[2024-10-16T09:55:37.924+0000] {sql.py:373} INFO - Rows affected: 1
[2024-10-16T09:55:37.936+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=exchange_rate_usd_kzt_dag, task_id=insert_rate, execution_date=20240520T000000, start_date=20241016T095537, end_date=20241016T095537
[2024-10-16T09:55:37.972+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2024-10-16T09:55:37.990+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-16T10:05:31.425+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-05-20T00:00:00+00:00 [queued]>
[2024-10-16T10:05:31.453+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-05-20T00:00:00+00:00 [queued]>
[2024-10-16T10:05:31.453+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T10:05:31.454+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2024-10-16T10:05:31.455+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T10:05:31.495+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): insert_rate> on 2024-05-20 00:00:00+00:00
[2024-10-16T10:05:31.508+0000] {standard_task_runner.py:55} INFO - Started process 11281 to run task
[2024-10-16T10:05:31.534+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'exchange_rate_usd_kzt_dag', 'insert_rate', 'scheduled__2024-05-20T00:00:00+00:00', '--job-id', '2467', '--raw', '--subdir', 'DAGS_FOLDER/exchange_rates/__init__.py', '--cfg-path', '/tmp/tmpwy5kn_3j']
[2024-10-16T10:05:31.536+0000] {standard_task_runner.py:83} INFO - Job 2467: Subtask insert_rate
[2024-10-16T10:05:32.037+0000] {task_command.py:389} INFO - Running <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-05-20T00:00:00+00:00 [running]> on host 63ee4fa64e74
[2024-10-16T10:05:32.246+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:205: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2024-10-16T10:05:32.418+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=exchange_rate_usd_kzt_dag
AIRFLOW_CTX_TASK_ID=insert_rate
AIRFLOW_CTX_EXECUTION_DATE=2024-05-20T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-05-20T00:00:00+00:00
[2024-10-16T10:05:32.420+0000] {sql.py:253} INFO - Executing: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'KZT', 443.83834098, '2024-05-20')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;
[2024-10-16T10:05:32.443+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T10:05:32.851+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T10:05:32.862+0000] {sql.py:364} INFO - Running statement: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'KZT', 443.83834098, '2024-05-20')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;, parameters: None
[2024-10-16T10:05:32.865+0000] {sql.py:373} INFO - Rows affected: 1
[2024-10-16T10:05:32.883+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=exchange_rate_usd_kzt_dag, task_id=insert_rate, execution_date=20240520T000000, start_date=20241016T100531, end_date=20241016T100532
[2024-10-16T10:05:32.929+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2024-10-16T10:05:32.949+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
